packages/usa-form/src/test/test-pattern/test-usa-form.twig

```diff
@@ -1,10 +1,11 @@
 <form class="usa-form">
   <div class="usa-character-count">
-    <div class="usa-form-group">
-      <label class="usa-label" for="with-hint-input">Character count</label>
+    <div class="usa-form-group {% if error_state %}usa-form-group--error{% endif %}">
```

Comment:
## `usa-form` and `usa-form-group`

>[!NOTE]
>While we *could* add the `usa-form-group--error` class to `<form class="usa-form">` elements, I believe we should avoid doing this and only add it to `usa-form-group` elements.
>
>Setting the error on the form element would add the error class to the entire form and not just the  form question that holds the error.

Reply:
This could be a useful comment in code.

Reply:
Comment added in [1723270](https://github.com/uswds/uswds/pull/5879/commits/172327098ca4986e9f6db9c35b33aea7cc1ebcf6)